### PR TITLE
Update the CHANGELOG, describing v1.4.7 changes

### PR DESCRIPTION
**Pull request by alexeyklyukin**
Update the CHANGELOG, describing v1.4.7 changes

### CODE REVIEW

**Comment by feikesteenbergen** [comment]
Shall we merge https://github.com/timescale/timescaledb-docker-ha/pull/277 first?

**Comment by alexeyklyukin** [comment]
> Shall we merge #277 first?

Let's merge it!

**Review by pgnickb — APPROVED** [review]

**Comment by feikesteenbergen** [comment]
Running a test to see if the latest promscale is installed ok using the binary packages, waiting for that ...

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,8 +9,11 @@ These are changes that will probably be included in the next release.
 
 ## [future release]
 
-* Upgrade Promscale extension to 0.5.3
+Describe the changes that will go into the next release here
 
+## [v1.4.7] - 2022-07-14
+
+* [Update OOMGuard](https://github.com/timescale/timescaledb-docker-ha/pull/279)
```
Comment (feikesteenbergen) [inline CHANGELOG.md:16]:
```suggestion
* [Update OOMGuard](https://github.com/timescale/timescaledb-docker-ha/pull/279)
* [Use binary packages for promscale and toolkit](https://github.com/timescale/timescaledb-docker-ha/pull/277)
```

**Review by feikesteenbergen — CHANGES_REQUESTED** [review]

**Review by feikesteenbergen — APPROVED** [review]

**Pull request merged**